doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1079,6 +1116,7 @@ Here are the syntax elements used by the various notation commands.
       | only printing
       | format @string {? @string }
       explicit_subentry ::= ident
+      | name
```

Comment:
The syntax blocks proceeded by `.. insertprodn` should be updated through doc_grammar rather than by hand.  If you don't do that, your change will disappear the next time doc_grammar updates the rsts.  Please run doc_grammar.